pathology/multiple_instance_learning/panda_mil_train_evaluate_pytorch_gpu.py

```diff
@@ -259,7 +207,7 @@ class LabelEncodeIntegerGraded(Transform):
 
     """
 
-    def __init__(self, num_classes, keys=["label"]):
+    def __init__(self, num_classes, keys=("label",)):
```

Comment:
If the transform works for `dict` data, could you please help change it to inherit from `MapTransform` base class?

Thanks.

Reply:
Sure, will do that.